drivers/bbram/bbram_npcx.c

```diff
@@ -35,6 +35,12 @@ static int get_bit_and_reset(const struct device *dev, int mask)
 	DRV_STATUS(dev) &= ~mask;
 #else
 	DRV_STATUS(dev) = mask;
+
+	if (IS_ENABLED(CONFIG_BBRAM_NPCX_STATUS_REG_WRITE_WORKAROUND)) {
+		uint8_t read_unused __attribute__((unused));
```

Comment:
could you use `uint8_t __unused unused;` here? It expands a bit differently but I think should work

https://github.com/zephyrproject-rtos/zephyr/blob/3d2059f9b66faf2b6cf4be15b2f4adc2f092f42b/include/zephyr/toolchain/gcc.h#L260

Reply:
Done.